### DISCUSSION TITLE
Is there a way to use the Broadcast kitten with hidden input?

### DISCUSSION THREAD

**Discussion by pr3c0g** [kovidgoyal/kitty#5868] [Q&A] [answered]
Hi all!

I would like to have the input passed via Broadcast, hidden on the Broadcast window itself.

An example use case would be passing sudo password to several windows at the same time, for when I'm ssh'ing to several different systems at the same time.

As an alternative, hiding the Broadcast window on demand or via a parameter, could also achieve this.

I still need to dive into the Broadcast kitten code more, but I was wondering if someone has this working already, and I couldn't find any topic about yet.

Thank you :) !

**Comment by kovidgoyal** [comment, marked as answer]
https://github.com/kovidgoyal/kitty/commit/69255f752576bb29cb5cdc2f7d8761fef1102cf6

**Reply by pr3c0g** [reply]
Sorry for taking so long to reply, just tried it out and it works perfectly!
Thank you!
